src/js/disability-benefits/routes.jsx

```diff
@@ -12,6 +12,10 @@ import AdditionalEvidencePage from './containers/AdditionalEvidencePage.jsx';
 import ClaimEstimationPage from './containers/ClaimEstimationPage.jsx';
 
 const routes = [
+  <Redirect
```

Comment:
Thanks for your question @jbalboni. Navigating to /disability-benefits/track-claims results in the following:
![screen shot 2017-05-09 at 10 06 20 am](https://cloud.githubusercontent.com/assets/16051172/25863118/671e1a80-349f-11e7-975c-2ad772e1d7ee.png)
---------------------------------------------------------------------------------------------
![screen shot 2017-05-09 at 10 06 39 am](https://cloud.githubusercontent.com/assets/16051172/25863151/86cbb342-349f-11e7-9b90-807f9d3b4f5a.png)
---------------------------------------------------------------------------------------------
![screen shot 2017-05-09 at 10 06 43 am](https://cloud.githubusercontent.com/assets/16051172/25863160/9050092c-349f-11e7-8a1a-9c3cf197c95f.png)